### PR TITLE
issue#1084 - Included the JIRA Issues VIew with Jira Plugin

### PR DESCRIPTION
**Pull request by riginoommen**
Issue - #1084
1. Included view for showing the JIRA issues along with activity stream.

jira-plugin-issues-view

#### :heavy_check_mark: Checklist

- [x] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [x] Screenshots of before and after attached (for UI changes)
- [x] Added or updated documentation (if applicable)

### CODE REVIEW

**Comment by riginoommen** [comment]
Requesting the Maintainers to please review he code.

File: plugins/frontend/backstage-plugin-jira/src/api/index.ts [hunk 1]
```diff
@@ -326,7 +348,7 @@ export class JiraAPI {
 
     let filterUrl = `streams=key+IS+${projectKey}`;
     if (ticketIds && (componentName || label)) {
-      filterUrl += `&streams=issue-key+IS+${ticketIds.join('+')}`;
+      filterUrl += `&streassms=issue-key+IS+${ticketIds.join('+')}`;
```
Comment (Xantier) [inline plugins/frontend/backstage-plugin-jira/src/api/index.ts:351]: is this the actual URL part or a typo?

File: .changeset/light-candles-develop.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-jira': major
```
[the remark this answers is not in the document]
Comment (riginoommen) [reply .changeset/light-candles-develop.md:2]: I felt a new feature like issues view is a feature and I may be major release

File: plugins/frontend/backstage-plugin-jira/src/api/index.ts (hunk 1 above)
Comment (riginoommen) [reply plugins/frontend/backstage-plugin-jira/src/api/index.ts:351]: its a typo correcting it

Comment (riginoommen) [reply plugins/frontend/backstage-plugin-jira/src/api/index.ts]: Corrected

File: .changeset/light-candles-develop.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (riginoommen) [reply .changeset/light-candles-develop.md:2]: I will update it as minor

[the remark this answers is not in the document]
Comment (riginoommen) [reply .changeset/light-candles-develop.md]: Updated to minor release

File: plugins/frontend/backstage-plugin-jira/src/api/index.ts [hunk 3]
```diff
@@ -296,7 +304,20 @@ export class JiraAPI {
...
+        assignee: {
+          displayName: index?.fields?.assignee?.displayName,
+          avatarUrl: index?.fields?.assignee?.avatarUrls['48x48'],
+        },
+        status: index?.fields?.status?.name,
+        priority: index?.fields?.priority,
+        created: index?.fields?.created,
+        updated: index?.fields?.updated,
+      };
+    }) as any;
```
Comment (Xantier) [inline plugins/frontend/backstage-plugin-jira/src/api/index.ts:320]: Let's specify a proper type for the `tickets` variable. This return from `map` doesn't need to be cast as `any`

Comment (riginoommen) [reply plugins/frontend/backstage-plugin-jira/src/api/index.ts]: Fixed this issue

**Review by Xantier — APPROVED** [review]

**Pull request merged**